files/build_templates/docker_image_ctl.j2

```diff
@@ -137,6 +137,9 @@ start() {
         -v /usr/share/sonic/device/$PLATFORM:/usr/share/sonic/platform:ro \
 {%- if docker_container_name != "database" %}
         -v /usr/share/sonic/device/$PLATFORM/$HWSKU:/usr/share/sonic/hwsku:ro \
+{%- endif %}
+{%- if docker_container_name == "iccpd" %}
+        -v /lib/modules:/lib/modules:ro \
```

Comment:
why do we need lib modules?

Reply:
MCLAG uses lib module ebtable_filter and ebtables

Reply:
can you document how ebtable_filter and ebtables are used in the design document, it needs to be review. 

besides, even mclag is uses ebtable, it does not mean we need to mount /lib/modules into the docker.

Reply:
@jianjundong, @shine4chen , can you please address this comment?  Please raise a separate PR for this. Thanks.

Reply:
We found that lib module ebtable_filter and ebtables are already installed in the docker default by someone, thus no need to do this here. These codes are removed by lguohan in PR '[docker-iccp]: do not mount kernel module into iccp container (#4372)'.